design/FY2021/NFP-CodeCompliantEquipmentDataset.md

```diff
@@ -0,0 +1,65 @@
+
+For years, chillers have been required in many building energy codes to have both a minimum full (expressed as a EER, COP, or kW/ton value) and part load efficiency (expressed as an IPLV value) which make them a good candidate for developing a methodology to develop such a set of curves.
+
+## E-mail and  Conference Call Conclusions ##
+
+## Overview ##
+The proposed changes add a new dataset, `CodeCompliantEquipment.idf`, that includes chiller performance curves to be use to meet the minimum requirements in ASHRAE 90.1-2019 and its Appendix G. We propose that the same dataset be improved and supplemented at a later date with additional performance curves for other equipment types (for instance: IEER-rated DX equipment).
+
+## Approach ##
+Generating chiller performances curves meeting both full and part load efficiency numbers require solving a largely underdetermined system of equations. The number of unknowns depend on the type of curves, which mostly depend on the algorithm used by the software and used in models. It also means that an infinite number of solutions exists. One way to solve this issue is to turn it into a constrained problem and use a simple genetic algorithm (GA) with a well-defined objective or cost function to determine potential solutions with physical significance. A GA can have a short computational time, be scalable, versatile, find solutions to problems with large number of parameters to problems with multiple local optima so it is a good candidate for this task.
+
+The implemented algorithm, which uses a GA, starts with a “seed” set of curves that can be user-modifiable, target full load, and part load efficiency as inputs was designed. The GA reproduces the process of natural selection on an initial population and subsequent generations. Individuals in the initial population are generated from the seed curves by randomly modifying the coefficient of each curve. An objective, fitness, or cost function is defined and is used to grade each individual solution on how close or how far their results  are from the target. Following the grading of each individual solution, some of the best performers are retained, others are discarded, and some are used to create new individual solutions during the “crossover” process by combining the best performing sets of curves together. Finally, some individuals are randomly modified during the “mutation” process to avoid local optima. The process is repeated (i.e., new generations are created) until an individual matching the targets is identified. Generation after generation, the average fitness improves until the target is met.
+
```

Comment:
And if there are edits there is an extra space here.

... far their results  are from the target.

Reply:
Thanks! I removed the extra space.